docs/user-guide/pods/init-container.md

```diff
@@ -162,7 +157,7 @@ reasons:
 ## Support and compatibilty
 
 A cluster with Kubelet and Apiserver version 1.4.0 or greater supports init
```

Comment:
Maybe say: 

> A cluster with Apiserver version 1.6.0 or greater supports init containers using the `spec.initContainers` field.  This field is also mirrored into alpha and beta annotations so that Kubelets version 1.3.0 or greater can execute init containers.